.eslintrc

```diff
@@ -62,7 +49,25 @@
           }
         ]
       }
-    ]
+    ],
+    "no-useless-concat": 0,
+    // react/function-component-definition affects 600+ instances. Temporarily disabled.
+    // ToDo: Enable gradually and apply component by component.
```

Comment:
also think we should discuss a little more if this is the correct direction we want to go.

Reply:
Agree.

Reply:
I don't think this rule feels super important to go back and turn on since it is just defining a code style for writing function components in the project versus a rule that is preventing us from bad patterns. Since it is an open source project this rule feels a little too restrictive without providing much benefit. Definitely curious to hear other people's opinions though.

Reply:
Thanks for the input and perspective @jcfilben.

Reply:
@MikeKingdom voiced similar thoughts, so it sounds like we have a consensus here.

I will update so that it is simply disabled.